docs/cppcx/collections-c-cx.md

```diff
@@ -12,9 +12,9 @@ The Windows Runtime defines the interfaces for collections and related types, an
 
 ![Diagram of C plus plus C X inheritance tree for collection types.](../cppcx/media/cppcxcollectionsinheritancetree.png)
 
-- The [Platform::Collections::Vector class](../cppcx/platform-collections-vector-class.md) resembles the [std::vector class](../standard-library/vector-class.md).
+- The [Platform::Collections::Vector Class](../cppcx/platform-collections-vector-class.md) resembles the [std::vector class](../standard-library/vector-class.md).
```

Comment:
This one is wrong. Our style is to use sentence casing. Class should be lowercase.